flink-python/dev/lint-python.sh

```diff
@@ -303,7 +352,7 @@ function install_environment() {
     # step-3 install python environment whcih includes
     # 3.5 3.6 3.7
     print_function "STEP" "installing python environment..."
```

Comment:
Move this print into the `if`? because the installation may not be called if it returns false.
Same for other commands.